src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -3307,27 +3307,62 @@ static void writeChordLines(const Chord* const chord, XmlWriter& xml, Notations&
 static void writeBreathMark(const Breath* const breath, XmlWriter& xml, Notations& notations, Articulations& articulations)
 {
     if (breath && ExportMusicXml::canWrite(breath)) {
+        String tagName;
+        String type;
+
         notations.tag(xml, breath);
         articulations.tag(xml);
         if (breath->isCaesura()) {
-            xml.tag("caesura");
+            tagName = u"caesura";
+            switch (breath->symId()) {
+            case SymId::caesuraCurved:
+                type = u"curved";
+                break;
+            case SymId::caesuraShort:
+                type = u"short";
+                break;
+            case SymId::caesuraThick:
+                type = u"thick";
+                break;
+            case SymId::caesuraSingleStroke:
+            case SymId::chantCaesura:
+                type = u"single";
+                break;
+            case SymId::caesura:
+            default:
+                type = u"normal";
```

Comment:
Why don't you leave the default an empty string? This is in MusicXML equivalent and wouldn't make problems with Mu3.

Reply:
That won't help much, that way older Mu versions would properly read that one particular caesura type, but still none of the other types.

Reply:
In Mu3, that is.

Reply:
OK. exporting as empty, at least then Mu3 reads this one without reporting corruptions